packages/react/src/patterns/sub-patterns/PictogramItem/PictogramItem.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import ContentItem from '../ContentItem/ContentItem';
+import {
+  settings as ddsSettings,
+  // markdownToHtml,
+} from '@carbon/ibmdotcom-utilities';
+//   import { ArrowRight20 } from '@carbon/icons-react';
+//   import { LinkWithIcon } from '../../../components/LinkWithIcon';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Converts markdown to html and returns object to be used in `dangerouslySetInnerHTML`
+ *
+ * @param {string} string string to be converted to html
+ * @returns {object} object with a __html key containing converted string
+ */
+//   const convertMarkdown = string => ({
+//     __html: markdownToHtml(string, {
+//       bold: false,
+//     }),
+//   });
+
+/**
+ * Content with pictogram component
+ *
+ * @param {object} props props object {title, copy, pictogram, link}
+ * @param {string} props.heading Content with pictogram component title property
+ * @param {string} props.copy Content with pictogram component copy property
+ * @param {*} props.Pictogram Content with pictogram component Pictogram component
+ * @returns {*} Content array with pictograms item JSX Component
+ */
+const PictogramItem = ({ heading, copy, Pictogram, cta }) => (
+  <div
+    data-autoid={`${stablePrefix}--pictogram-item`}
+    className={`${prefix}--pictogram-item`}>
+    <div className={`${prefix}--pictogram-item__row`}>
+      <div className={`${prefix}--pictogram-item__pic-col`}>
+        <Pictogram
+          data-autoid={`${stablePrefix}--pictogram-item_pictogram`}
+          className={`${prefix}--pictogram-item__pictogram`}
+          viewBox="8 8 32 32"
```

Comment:
Is this not defined in the SVG itself?

Reply:
The `Pictogram` is the SVG imported by the adopter.

Reply:
So we're passing in the `SVG`, and not the Pictogram component coming from Carbon?

[Pictogram Library](https://www.carbondesignsystem.com/guidelines/pictograms/library)
[@carbon/pictograms](https://www.npmjs.com/package/@carbon/pictograms)
[@carbon/pictograms-react](https://www.npmjs.com/package/@carbon/pictograms-react)

Reply:
Actually, I've misunderstood, this pictogram is received by a prop, the user needs to import the Pictogram and pass it to the component

Reply:
Ok, so if we're pulling it from @carbon/pictograms-react — as we should — then I believe the viewBox is already included. Am I seeing that correctly?

https://raw.githubusercontent.com/carbon-design-system/carbon/master/packages/pictograms/svg/agriculture.svg

Reply:
The default viewbox does not match our sizing requirements, that's why I overwrote it.